### DISCUSSION TITLE
如何退款,  买了两年, 结果用的时候发现不是我想要的

### DISCUSSION THREAD

**Discussion by hyysgit** [getlantern/lantern-forum-cn#496] [Q&A]
想着蓝灯也是老牌了, 就一口气买了两年, 发现用的时候, 我的 苹果13 无法使用, 每次打开 都是闪退.
然后用安卓体验了一下, 无法切换地区, 对我来说 不适合,  买的当天我就给 support@lantern.jitbit.com 提交 邮件需要退款,
就一个自动回复,  四天过去了都还没一点消息
 ticket 号:     #72120764

![%5IGBNO {F14172TSNOT0~Y](https://github.com/getlantern/lantern-forum-cn/assets/47076252/4ce07613-225d-4d6d-bd6b-d0d3cac9bf8c)

**Comment by MaxMKMax** [comment]
@hyysgit 您好.! 首先, 想请问您一下您是否有下载最新版本呢? 最新版本应该不会闪退了.
下载链接:
https://github.com/getlantern/lantern

如果真的不行, 我帮您申请退款吧. 请您提供我账号邮箱和支付单据.

**Reply by MaxMKMax** [reply]
@hyysgit 好的. 已经帮您提交上去了. 请您等待 10-14 天来完成退款.
